keps/sig-storage/3333-retroactive-default-storage-class/README.md

```diff
@@ -581,6 +581,7 @@ Major milestones might include:
 -->
 - 1.25: initial version
 - 1.26: beta
+- 1.28: GA
```

Comment:
Can't comment on unchanged lines, so commenting here:

L393: Were upgrade->rollback->upgrade tested?
Can you please describe the test that were run and the findings?
[https://github.com/kubernetes/enhancements/pull/3658/files is one great example of it]

L406: was the metric added? Can you please link its name?

Reply:
Yes, as a comment in https://github.com/kubernetes/enhancements/issues/3333#issuecomment-1282393389

Reply:
Sure, added.